### PR TITLE
Test for missing group.id

### LINKED ISSUE #8471: RM: Error messages for bad configuration
Ensure that we have useful error messages which explain the problem when the user misconfigures the reactive messaging kafka connector.

We at least need sensible messages for:
* No bootstrap server provided
* ~No topic provided~ Topic will default to channel name https://github.com/OpenLiberty/open-liberty/issues/8604
* Any connector-specific (i.e. not passed through to the kafka client) properties with invalid values

### PR DESCRIPTION
**Pull request by tevans78**
#build 
fixes #8471

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops [hunk 1]
```diff
@@ -50,6 +50,11 @@ kafka.output.error.signal.CWMRX1004E=CWMRX1004E: The Reactive Messaging Kafka Ou
 kafka.output.error.signal.CWMRX1004E.explanation=The Kafka Outgoing Connector received an error signal that prevents normal operation of Reactive Messaging.
 kafka.output.error.signal.CWMRX1004E.useraction=Review the server message.log file and FFDC logs to identify the problem.
 
+# The group.id property was not set in the configuration for this channel. Set the {0} property in one of the MicroProfile Config Sources for this application.
+kafka.groupid.not.set.CWMRX1005E=CWMRX1005E: The group.id property was not set in the configuration for this channel. Set the {0} property in one of the MicroProfile Config Sources for this application.
+kafka.groupid.not.set.CWMRX1005E.explanation=The Kafka Consumer Group ID must be set to allow Reactive Messaging to function properly.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: Change
"The Kafka Consumer Group ID must be set to allow Reactive Messaging to function properly."
To
"The Kafka consumer group ID must be set to allow Reactive Messaging to function properly."

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops [hunk 2]
```diff
@@ -50,6 +50,11 @@ kafka.output.error.signal.CWMRX1004E=CWMRX1004E: The Reactive Messaging Kafka Ou
...
+kafka.groupid.not.set.CWMRX1005E.useraction=Set the property into one of the MicroProfile Config Sources for this application e.g. the microprofile-config.properties file.
```
[the remark this answers is not in the document]
Comment (tevans78) [reply dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: Will make these changes in a follow up PR as I need to merge this now. Thank you.

[the remark this answers is not in the document]
Comment (tevans78) [reply dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: I opened #8708 to do the updates

**Comment by tevans78** [comment]
Personal Build was not green but the Reactive Messaging tests all passed which is good enough for this.

**Pull request merged**